src/cpu/CMakeLists.txt

```diff
@@ -38,6 +38,17 @@ if((DNNL_TARGET_ARCH STREQUAL "X64") OR (DNNL_TARGET_ARCH STREQUAL "AARCH64"))
     endforeach()
 endif()
 
+if(DNNL_TARGET_ARCH STREQUAL "PPC64")
+    file(GLOB FILES_REQUIRED_OPT
+        ${CMAKE_CURRENT_SOURCE_DIR}/gemm/*.[ch]pp
+        )
+    foreach(SOURCE_FILE ${FILES_REQUIRED_OPT})
+        list(APPEND SOURCES "${SOURCE_FILE}")
+    endforeach()
```

Comment:
I feel this is a duplicate of line 21.

Reply:
I am not an expert in the use of these cmake things, but I think this is here for the single purpose of guaranteeing the compile flags "-O3 and -funroll-loops" would be added to the compilation of our code.  But thanks for pointing me at this, because we need to add more code to this list!